io-engine/src/grpc/v0/mayastor_grpc.rs

```diff
@@ -249,6 +249,22 @@ impl From<LvsError> for tonic::Status {
                 Errno::EMEDIUMTYPE => Status::aborted(e.to_string()),
                 _ => Status::internal(e.to_string()),
             },
+            LvsError::RepResize {
+                source, ..
+            } => match source {
+                Errno::ENOSPC | Errno::ENOMEM => Status::resource_exhausted(
+                    "Out of space/memory: replica resize failed".to_string(),
```

Comment:
You can use e.verbose() on each of these to include the error chain, rather than manually doing it as there's ambiguity here, example "Out of space or memory" ?